frontend/public/components/utils/rbac.tsx

```diff
@@ -87,10 +89,15 @@ type RequireCreatePermissionProps = {
   children: React.ReactNode;
 };
 
-export const asAccessReview = (kindObj: K8sKind, obj: K8sResourceKind, verb: K8sVerb): AccessReviewResourceAttributes => ({
-  group: kindObj.apiGroup,
-  resource: kindObj.plural,
-  name: obj.metadata.name,
-  namespace: obj.metadata.namespace,
-  verb,
-});
+export const asAccessReview = (kindObj: K8sKind, obj: K8sResourceKind, verb: K8sVerb): AccessReviewResourceAttributes => {
+  if (!obj){
```

Comment:
return null if the obj is null

Reply:
If you don't return null here, it will make an entirely different, unintended access review check. That is my original concern with the change :)

Reply:
ah sorry, my bad. Fixed